core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20161202_add_json_attributevalue.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet id="add_json_entity_attribute" author="hussein" logicalFilePath="dummy">
+        <sql>
+            ALTER TABLE ENTITY_ATTRIBUTE ADD COLUMN value_json LONGTEXT;
+        </sql>
+        <sql>
+            ALTER TABLE ENTITY_ATTRIBUTE_SCRATCH ADD COLUMN value_json LONGTEXT;
+        </sql>
+    </changeSet>
+
+    <changeSet id="add_json_workspace_attribute" author="hussein" logicalFilePath="dummy">
+        <sql>
+            ALTER TABLE WORKSPACE_ATTRIBUTE ADD COLUMN value_json LONGTEXT;
+        </sql>
+        <sql>
+            ALTER TABLE WORKSPACE_ATTRIBUTE_SCRATCH ADD COLUMN value_json LONGTEXT;
+        </sql>
```

Comment:
`alter table` does not really participate in a transaction because it can't be rolled-back. In fact, the database will automatically commit any existing transaction before executing the `alter table` statement.

The other option is to use a `<rollback>` block, but that has its own issues (what do you do if rollback fails part way through?).

BTW, is there a reason to use hand-written DDL here instead of Liquibase's `<addColumn>`? (http://www.liquibase.org/documentation/changes/add_column.html)